### ISSUE TITLE
Incompatible with IC2 Classic

### ISSUE THREAD

**Issue by Nocty201** [fewizz/Electro-Magic-Tools#12]
Test environment: MC1.7.10, Forge 1448, IC2 Classic 1.1.2.1 (http://forum.industrial-craft.net/index.php?page=Thread&threadID=11201), TC 4.2.3.5, and EMT 1.2.2

Crashes on startup, log here: https://paste.ee/p/i7GC6

" java.lang.NoClassDefFoundError: ic2/core/init/InternalName "

**Comment by MonzterAssassin** [comment]
This addon is a plugin for Thaumcraft and IC2 Experimental. Not IC2 Classic. Check the description before you report the error of a mod not designed to work with that.

**Comment by Nocty201** [comment]
Yes, I get that. ;)

At the very least, it's something for others to find so they know it's not going to work.

Sadly, given this addon is no longer maintained (or so I've been led to believe), it's not likely to be added as a feature, either.

**Comment by MonzterAssassin** [comment]
Yes alass but perhaps fewizz realizes some of us like the unequity of IC2
and EMT because some of us actually played IC2 when it was one of the first
tech mods! (Like ME! :D)

**Comment by grydian** [comment]
I actually am eagerly awaiting this feature so that I can use this mod in my mod pack.  I love ic2 classic and really would love to use this mod with it.
